scripts/oes-data-migration-scripts/migration_v3.12.x_to_v4.0.x.py

```diff
@@ -491,8 +492,10 @@ def close_connections():
         audit_conn.close()
         if audit_conn is not None:
             audit_conn.close()
-        if spindb.is_connected():
-            spindb.close()
+        if spindb_orca.is_connected():
```

Comment:
similar change is missing inside "commit_transactions()"

Reply:
updated